openmetadata-ui/src/main/resources/ui/src/components/UserDetails/UserDetails.tsx

```diff
@@ -174,7 +174,7 @@ const UserDetails = ({
           totalCount={userPaging.total}
         />
       )}
-      {!isUndefined(deletingUser) && (
+      {/* {!isUndefined(deletingUser) && (
```

Comment:
Let's remove the code block if it is not needed

Reply:
Thanks for catching it!